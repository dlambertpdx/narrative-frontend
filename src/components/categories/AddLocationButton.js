import React from 'react';
import PropTypes from 'prop-types';
import EditFormModalLocation from './EditFormModalLocation';

const AddLocationButton = ({ show, hideModal, showModal }) => {
  return (
    <>
      <EditFormModalLocation show={show} hideModal={hideModal}></EditFormModalLocation>
      <div>
        <button type='button' onClick={showModal}>Add New</button>
      </div>
    </>
  );
};

AddLocationButton.propTypes = {
  show: PropTypes.bool,
  hideModal: PropTypes.func,
  showModal: PropTypes.func
};

export default AddLocationButton;
