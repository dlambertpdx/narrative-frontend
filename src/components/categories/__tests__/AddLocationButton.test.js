import React from 'react';
import { shallow } from 'enzyme';
import AddLocationButton from '../AddLocationButton';

describe('AddLocationButton component', () => {
  it('renders an add button', () => {
    const wrapper = shallow(<AddLocationButton handleAddClick={() => true} />);
    expect(wrapper).toMatchSnapshot();
  });
});