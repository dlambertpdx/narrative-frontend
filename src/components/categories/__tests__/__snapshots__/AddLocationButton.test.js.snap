// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddLocationButton component renders an add button 1`] = `
<Fragment>
  <EditFormModalLocation />
  <div>
    <button
      type="button"
    >
      Add New
    </button>
  </div>
</Fragment>
`;
